downstream/modules/aap-on-azure/proc-azure-configure-ad-sso.adoc

```diff
@@ -69,23 +69,23 @@ Add the key _(Application (client) ID)_ and value (_Value_) of the secret that y
 
 . Open the {ControllerName} console in a web browser.
 . Click menu:Settings[Azure AD settings].
-. Click *Edit*.
+. Click btn:[Edit].
 . Enter the information for the secret that you generated in Azure AD:
   . In *Azure AD OAuth2 Key*, paste the _Application (client) ID_.
   . In *Azure AD OAuth2 Secret*, paste the secret _Value_.
-. Click *Save*.
+. Click btn:[Save].
 
 .Adding Azure Credentials to Automation controller
 
 . Open the {ControllerName} in a web browser.
-. Under *Resources*, click *Credentials*.
-. Click *Add* to open the *Create New Credentials* page.
+. Under *Resources*, click btn:[Credentials].
```

Comment:
Not sure this is a button. I think it might be a menu cascade selection. If so, suggest:

Select menu:Resources[Credentials].

You could merge steps 2 and 3 to become:
Select menu:Resources[Credentials] and click btn:[Add] to open the *Create New Credentials* page.